include/nbl/asset/IRenderpassIndependentPipeline.h

```diff
@@ -410,7 +410,7 @@ struct SColorAttachmentBlendParams
         memcpy(bf_dst, &bf, serializedSize());
     }
 } PACK_STRUCT;
-static_assert(sizeof(SColorAttachmentBlendParams)==6u, "Unexpected size of SColorAttachmentBlendParams (should be 6)");
+static_assert(sizeof(SColorAttachmentBlendParams)==5u, "Unexpected size of SColorAttachmentBlendParams (should be 5)");
```

Comment:
what changed?

Reply:
nothing, bitfields works differently on gcc, I'll make sure it builds on msvc and gcc later. + I have no idea why this built for cyprian

Reply:
> I'll make sure it builds on msvc and gcc later. 
Essential

whenever we have a  `static_assert` on a struct its because we intend to dump it bytewise into a `.nbl` blob (previously `.baw`)

AFAIK for bitfields we custom serialize/deserialize

BUT this struct having a different size across compilers is a problem, because it gets used in a class, which changes the bytesize of the class which then changes the size of the fixed blob portion and you have a problem.

So either fix the oredering or something or abandon bitfield use here

Reply:
> whenever we have a static_assert on a struct its because we intend to dump it bytewise into a .nbl blob (previously .baw)

I have serialize methods for those structs so I think those static asserts are pointless now

> this struct having a different size across compilers is a problem, because it gets used in a class, which changes the bytesize of the class which then changes the size of the fixed blob portion and you have a problem

blob will have fixed size, those structs doesn't have to as long as we have serialize methods

Reply:
@AnastaZIuk make note of this for the future